docs/conf.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
```

Comment:
FYI this doc was auto-generated when running `sphinx-quickstart` - I cleaned up some of the extraneous comments/unused config values.